### PR TITLE
Added new property to hide oncoprint menu settings

### PR DESCRIPTION
**Pull request by dianab0**
Added new portal.properties property to hide oncoprint
menu settings "cBioPortal>=" and "COSMIC>=";
Added in the comments of disabled_tabs property the comparison word,
as the comparison tab can be disabled as well but was not in the list.

### CODE REVIEW

File: portal/src/main/webapp/config_service.jsp [hunk 1]
```diff
@@ -59,6 +59,7 @@
             "priority_studies",
             "show.hotspot",
             "show.oncokb",
+            "show.oncokb.cbiocosmic",
```
Comment (pvannierop) [inline portal/src/main/webapp/config_service.jsp:62]: Naming is incorrect. Also, please separate cbioportal and cosmic options:

```suggestion
            "show.cbioportal",
             "show.cosmic",
```

File: src/main/resources/portal.properties.EXAMPLE [hunk 2]
```diff
@@ -194,6 +194,9 @@ oncokb.token=
 # Enable merging of OncoKB icons by default
 # oncokb.merge_icons_by_default=true
 
+# Enable OncoKB menu settings "cBioPortal>=" and "COSMIC>=" (true, false)
+show.oncokb.cbiocosmic=true
```
Comment (pvannierop) [inline src/main/resources/portal.properties.EXAMPLE:198]: Correct names and separate cbioportal and cosmic.

File: docs/portal.properties-Reference.md [hunk 3]
```diff
@@ -301,6 +301,10 @@ A private token is required to access the OncoKB Data (for details see the secti
 ```
 oncokb.token=
 ```
+OncoKB menu settings "cBioPortal>=" and "COSMIC>=" can be hidden by turning off the following property (default: true):
```
Comment (pvannierop) [inline docs/portal.properties-Reference.md:304]:
```suggestion
"cBioPortal>=" and "COSMIC>=" driver annotation sources in the settings menu of Results View can be hidden by turning off the following property (default: true):
```

File: docs/portal.properties-Reference.md [hunk 4]
```diff
@@ -301,6 +301,10 @@ A private token is required to access the OncoKB Data (for details see the secti
...
+```
+show.oncokb.cbiocosmic=true|false
```
Comment (pvannierop) [inline docs/portal.properties-Reference.md:306]: Naming is incorrect (see below).

File: src/main/resources/portal.properties.EXAMPLE [hunk 5]
```diff
@@ -194,6 +194,9 @@ oncokb.token=
 # Enable merging of OncoKB icons by default
 # oncokb.merge_icons_by_default=true
 
+# Enable OncoKB menu settings "cBioPortal>=" and "COSMIC>=" (true, false)
```
Comment (pvannierop) [inline src/main/resources/portal.properties.EXAMPLE:197]:
```suggestion
# Enable "cBioPortal>=" and "COSMIC>=" driver annotation sources in the settings menu of Results View (true, false)
```

**Review by pvannierop — CHANGES_REQUESTED** [review]
Please update nomenclature and separate the _cBioPortal_ and _COSMIC_ driver annotation sources. The menu is not the _OncoKB menu_, but rather an annotation sources menu. _OncoKB_ is an entry there equivalent to COSMIC, Hotspots, etc.

File: portal/src/main/webapp/config_service.jsp (hunk 1 above)
Comment (dianab0) [reply portal/src/main/webapp/config_service.jsp:62]: fixed

File: src/main/resources/portal.properties.EXAMPLE (hunk 2 above)
Comment (dianab0) [reply src/main/resources/portal.properties.EXAMPLE:198]: fixed

File: docs/portal.properties-Reference.md (hunk 3 above)
Comment (dianab0) [reply docs/portal.properties-Reference.md:304]: fixed

File: src/main/resources/portal.properties.EXAMPLE (hunk 5 above)
Comment (dianab0) [reply src/main/resources/portal.properties.EXAMPLE:197]: fixed

File: docs/portal.properties-Reference.md (hunk 4 above)
Comment (dianab0) [reply docs/portal.properties-Reference.md]: fixed

File: portal/src/main/webapp/config_service.jsp [hunk 6]
```diff
@@ -59,6 +59,8 @@
             "priority_studies",
             "show.hotspot",
             "show.oncokb",
+            "show.cbioportal",
+	    "show.cosmic",
```
Comment (dippindots) [inline portal/src/main/webapp/config_service.jsp:63]: Could we align this with the rest?

**Review by dippindots — COMMENTED** [review]
@dianab0 Just one minor comment about the text aligning, we can merge after an update.

File: portal/src/main/webapp/config_service.jsp (hunk 6 above)
Comment (dippindots) [inline portal/src/main/webapp/config_service.jsp:63]:
```suggestion
            "show.cbioportal",
            "show.cosmic",
```

**Review by pvannierop — APPROVED** [review]
Thank you!

File: portal/src/main/webapp/config_service.jsp (hunk 6 above)
Comment (dianab0) [reply portal/src/main/webapp/config_service.jsp]: Fixed

**Pull request merged**